tests/test_functional.py

```diff
@@ -0,0 +1,399 @@
+# Copyright (c) 2014 Google, Inc.
+# Copyright (c) 2014-2016 Claudiu Popa <pcmanticore@gmail.com>
+
```

Comment:
alright alright, this will be a first.
since my shopify email will be deprecated soon, should I put `data-acceleration@shopify.com` ?